docs/getting-started-guides/mesos.md

```diff
@@ -0,0 +1,297 @@
+## Getting started with Kubernetes on Mesos
+
+<!-- TODO: Update, clean up. -->
+
+Mesos allows dynamic sharing of cluster resources between Kubernetes and other first-class Mesos frameworks such as [Hadoop][1], [Spark][2], and [Chronos][3].
+Mesos ensures applications running on your cluster are isolated and that resources are allocated fairly.
```

Comment:
Maybe clarify to "Mesos ensures applications from different frameworks running on your cluster..."
I think it's fine to leave as-is, but since Kubernetes also provides isolation and resource allocation, it might be good to clarify what kind of isolation/resource allocation you're talking about.